techhive/server.js

```diff
@@ -39,6 +44,60 @@ app.get('/internships/:id', async (req, res) => {
   }
 });
 
+// Route to get all users
+app.get('/users', async (req, res) => {
+  try {
+    const users = await User.findAll();
+    res.json(users);
+  } catch (err) {
+    res.status(500).json({ message: err.message });
+  }
+});
+
+// Route to get a user by id
+app.get('/users/:id', async (req, res) => {
+  try {
+    const user = await User.findByPk(req.params.id);
+    if (user) {
+      res.json(user);
+    } else {
+      res.status(404).json({ message: 'User not found' });
+    }
+  } catch (err) {
+    res.status(500).json({ message: err.message });
+  }
+});
+
+// Route to get all posts, with associated users
```

Comment:
Assuming posts here are internship postings, what exactly are associated users? 
How do we define the association? Are they, for example, users that have saved/applied/recommended for a given internship? 

This context would be helpful to include in your PR description